app/features-json/graph_ql/queries/project_query.rb

```diff
@@ -0,0 +1,20 @@
+require_relative "../types/project_type"
+
+module FastlaneCI
+  module Query
+    Projects = GraphQL::ObjectType.define do
+      name "Projects"
+      field :projects, types[Types::ProjectType] do
+        argument(:id, types.String)
+        description "Get a list of Projects"
```

Comment:
Should the description be for the ObjectType, rather than the field?

Reply:
So the matter with this query is an `Object` per sé, so to diff the `Object` that mirrors the `Model` than the `Object` used as a `Query` I added that description, that being said, is totally optional and I can change it to whatevs

Reply:
if the graphql endpoint shows the description when you select projects, all is good :)